<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-ORI8JR52" name="demo.guru99.com.runner.RunCukes" tests="1" failures="0" timestamp="2021-01-16T17:52:25 IST" time="67.304" errors="1">
  <testcase name="parallelRun" time="67.304" classname="demo.guru99.com.runner.RunCukes">
    <error type="courgette.runtime.CourgetteTestFailureException" message="There were failing tests. Refer to the Courgette html report for more details.">
      <![CDATA[courgette.runtime.CourgetteTestFailureException: There were failing tests. Refer to the Courgette html report for more details.
]]>
    </error>
  </testcase> <!-- parallelRun -->
  <system-out/>
</testsuite> <!-- demo.guru99.com.runner.RunCukes -->
